Loaded dependency [numpy/1.22.3-python-3.9.11-openblas-0.3.19]: openblas/0.3.19
Loaded dependency [python3/3.9.11]: gcc/10.3.0-binutils-2.36.1
Loaded dependency [numpy/1.22.3-python-3.9.11-openblas-0.3.19]: python3/3.9.11
Loaded dependency [scipy/1.7.3-python-3.9.11]: numpy/1.22.3-python-3.9.11-openblas-0.3.19
Loaded module: scipy/1.7.3-python-3.9.11

Loading scipy/1.7.3-python-3.9.11
  Loading requirement: openblas/0.3.19 gcc/10.3.0-binutils-2.36.1 python3/3.9.11
    numpy/1.22.3-python-3.9.11-openblas-0.3.19
Loaded module: cuda/11.7
/work3/s204117/News-Recommendations/DataIterator.py:138: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  self.user_data['history_length'][self.user_data['history_length'] > 50] = 50
/work3/s204117/News-Recommendations/DataIterator.py:138: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  self.user_data['history_length'][self.user_data['history_length'] > 50] = 50
